components/__tests__/Themed-test.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { render } from '@testing-library/react-native'
+import { Text, View } from '../Themed'
+
+describe('Themed component tests', () => {
+  it('renders themed text correctly', () => {
+    const tree = render(<Text />).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+
+  it('renders themed View correctly', () => {
+    const tree = render(<View />).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
```

Comment:
I don't really love snapshot testing because it doesn't really give you much confidence. Its really just checking to see if the file changed which isn't very useful imho. Can we stick to just React Native Testing Library testing? :)